.github/workflows/tests.yaml

```diff
@@ -177,6 +180,23 @@ jobs:
             git.commit-sha=${{ github.sha }}
           cache-from: type=gha,scope=cloudserver
           cache-to: type=gha,mode=max,scope=cloudserver
+
+      - name: Build and push federation image
+        uses: docker/build-push-action@v5
+        with:
+          push: true
+          context: images/federation
+          provenance: false
+          build-args:
+            CLOUDSERVER_VERSION=${{ github.sha }}
+          tags: |
+            ghcr.io/${{ github.repository }}:${{ github.sha }}-federation
```

Comment:
shall we seize this opportunity to use a version before the sha?

Reply:
no completely trivial (need to compute the version) : will be done separately